setup.py

```diff
@@ -76,10 +81,11 @@ def get_all_extras() -> Dict:
     "pyyaml>=4.2b1",
     "requests>=2.22.0",
     "python-dotenv>=0.14.0",
+    "urllib3==1.25.8",
```

Comment:
Can we do `>=`?

Reply:
I think so. Also, we only use `urllib3` for parsing URLs.

![image](https://user-images.githubusercontent.com/9467617/107778997-0f316b00-6d45-11eb-89d0-3b79abea29f9.png)

Reply:
can we use urllib instead? https://docs.python.org/3/library/urllib.parse.html